ofl/ballet/METADATA.pb

```diff
@@ -0,0 +1,18 @@
+name: "Ballet"
+designer: "Omnibus-Type, Maximiliano Sproviero"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2020-09-24"
+fonts {
+  name: "Ballet"
+  style: "normal"
+  weight: 400
+  filename: "Ballet[opsz].ttf"
+  post_script_name: "Ballet-Regular"
+  full_name: "Ballet Regular"
+  copyright: "Copyright 2020 The Ballet Project Authors (https://github.com/Omnibus-Type/Ballet)"
+}
```

Comment:
This is missing the axes data

Reply:
there is no weight axis and the opsz is took of because not served by GF

Reply:
`opsz` is served by GF API (you can see some listed in fonts.google.com/variablefonts) but not surfaced for direct discovery in the main fonts.google.com UI.

Reply:
Ok cool, fixed here https://github.com/google/fonts/pull/2716